### PR TITLE
add ability to change network and use custom rpc on each one

### PR DESCRIPTION
**Pull request by zielvna**

### CODE REVIEW

**Review by p6te — COMMENTED** [review]
after choosing mainnet network,  select rpc doesn't work. it is settled to custom only with testnet 
image

File: src/store/consts/static.ts [hunk 1]
```diff
@@ -47,62 +47,113 @@ export const TokenAirdropAmount = {
   USDC: 50000000n
 }
 
-export const FaucetTokenList = {
-  BTC: TESTNET_BTC_ADDRESS,
-  ETH: TESTNET_ETH_ADDRESS,
-  USDC: TESTNET_USDC_ADDRESS
+export const getFaucetTokenList = (network: Network) => {
```
Comment (p6te) [inline src/store/consts/static.ts:50]: do we need faucet token list for mainnet as well?

**Comment by p6te** [comment]
when you choose tokens to swap, then switch network on position list, and go back to swap. tokens changed addresses and can't be readed from URL. maybe we can try to read them properly, because it works, when you stay on swap page. or just clear url to not misunderstand the user. 
[Nagranie ekranu z 30.08.2024 10:12:29.webm](https://github.com/user-attachments/assets/0b7bc48f-c02b-4914-a4f4-de810743270a)

**Review by p6te — APPROVED** [review]

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
We should hide faucet on "Mainnet"
image

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Probably the mainnet rpc has a different name:
image

**Review by wojciech-cichocki — APPROVED** [review]

**Pull request merged**